optimizely/client/client.go

```diff
@@ -76,9 +76,17 @@ func (o *OptimizelyClient) GetEnabledFeatures(userContext entities.UserContext)
 
 	defer func() {
 		if r := recover(); r != nil {
-			errorMessage := fmt.Sprintf(`optimizely SDK is panicking with the error "%s"`, string(debug.Stack()))
-			err = errors.New(errorMessage)
+			switch t := r.(type) {
```

Comment:
We need this in `IsFeatureEnabled` as well as the `GetFeatureVariable*` methods.